app/presenters/plans_base_presenter.rb

```diff
@@ -13,14 +13,18 @@ def initialize(service:, collection:, params: {})
 
   attr_reader :service, :collection, :pagination_params, :search, :sorting_params
 
-  def paginated_plans
-    @paginated_plans ||= plans.paginate(pagination_params)
-  end
-
   def plans
     @plans ||= collection.not_custom
-                         .reorder(sorting_params)
-                         .scope_search(search)
+                         .reorder('name ASC')
```

Comment:
Instead of reordering by `name asc` to later reorder by sorting_params (which btw default value is name asc)
I would create a different collection for the default_plans select and make plans private.
```ruby
private

def plans
  @plan ||= collection.not_custom
end
```